pkg/ycsb/db.go

```diff
@@ -70,6 +70,16 @@ type DB interface {
 	Delete(ctx context.Context, table string, key string) error
 }
 
+// WrappedDB gives some combined operations based on basic DB Operations.
+type WrappedDB interface {
```

Comment:
bad name

Reply:
any suggested?

Reply:
I am confused why do you want to provide this Interface?